gensim/similarities/docsim.py

```diff
@@ -444,8 +448,20 @@ def save(self, fname=None, *args, **kwargs):
         self.close_shard()
         if fname is None:
             fname = self.output_prefix
+
         super(Similarity, self).save(fname, *args, **kwargs)
 
+    @classmethod
+    def load(self, fname, mmap=None):
+
+        obj = super(Similarity, self).load(fname, mmap)
+        if isinstance(obj,Similarity):
+            if obj.output_prefix is not fname:
+                obj.output_prefix = fname
```

Comment:
yes, I think we discussed this only [here](https://github.com/RaRe-Technologies/gensim/pull/1226#discussion_r125277354). Correct me if I'm wrong